google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -383,6 +410,22 @@ private void maybeWaitForInflightQuota() {
     inflightWaitSec.set((System.currentTimeMillis() - start_time) / 1000);
   }
 
+  @VisibleForTesting
+  static long calculateSleepTime(long retryCount) {
+    return (long) Math.pow(5, retryCount);
```

Comment:
Are we backing off too much? Maybe base 2 with a max interval of say 1 min.

Reply:
added a max, but it's millisecond, do we still think 5 is too much?

Reply:
We have a total time out of retry 5 minutes. So we end up can only retry 7 times? As long as there is a max, I am OK.

Reply:
Let's change to 2 to retry more times